src/Core/SecretsManager/Models/Data/ProjectPermissionDetails.cs

```diff
@@ -4,6 +4,7 @@ namespace Bit.Core.SecretsManager.Models.Data;
 
 public class ProjectPermissionDetails : Project
 {
+    public Project Project;
```

Comment:
`ProjectPermessionDetails` is already a `Project`. Why are we adding this?

Reply:
Good catch, the intention was to match `SecretPermissionDetails` with this update for consistency. 

I forgot to remove the inheritance when altering the model.

Should be fixed in https://github.com/bitwarden/server/pull/2822/commits/1d99670bd9b5555254a11b719597a7833f5270f2